### ISSUE TITLE
Credit invoice with different template

### ISSUE THREAD

**Issue by EnMaster** [InvoicePlane/InvoicePlane#763]
In the most country of the Europe, credit invoice had different numeration and different layout because credit invoice was different from invoice.
It would be very useful if credit invoice had different html template

**Comment by clockwiseq** [comment]
I'm not fully sure I understand this one.  So you just want the ability to have different invoice types (sales/credit) and then have the ability to select a different template for each type?

**Comment by EnMaster** [comment]
> I'm not fully sure I understand this one. So you just want the ability to have different invoice types (sales/credit) and then have the ability to select a different template for each type?

Yes, sales and credit invoice need different template. At the moment i have create a workaround based on GROUP_TYPE, but is not the best.

**Comment by Verony-makesIT** [comment]
@clockwiseq, @nielsdrost7,  
Can't this Feature Request be extended with the issue on the forum ([Credit invoice date not correct](https://community.invoiceplane.com/t/topic/10660)) and my forum request [(Credit invoice template feature developed](https://community.invoiceplane.com/t/topic/10644))?
If this is ok for you then @Laz61 (in the forum) also does not need to create another Feature Request and then everything (related to Credit note/invoice) can be collected in this  Feature Request.

As @EnMaster points out, a Credit Note/Invoice has different numbering than an invoice and indeed also a ( slightly) different layout than an invoice.
In essence (at least in Belgium), a Credit Note/Invoice is no different than an invoice with a negative amount and slightly different (_**mandatory**_) entries.
However, I integrated the different templates into 1 template (invoice and Credit Note/Invoice) and took into account the differences between these two documents.
@EnMaster, can you do tell me what (layout) differences or entries there are in your country (Italy) between an invoice and a credit note/invoice?

> As you guys can read in my FR on the forum I have already developed this for myself and can submit a PR of this within a few days....
**Please let me know what you guys think about this ASAP**.

**Comment by nielsdrost7** [comment]
Sure, go ahead @Verony-makesIT and do your thing, make your pull-request

**Comment by nielsdrost7** [comment]
@Verony-makesIT see my remarks to your question in your pull-request
